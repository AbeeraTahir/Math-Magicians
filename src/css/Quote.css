h2 {
  position: fixed;
  top: 48%;
  left: 10%;
  right: 10%;
}
