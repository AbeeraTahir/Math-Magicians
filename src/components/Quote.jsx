import React from 'react';
import '../css/Quote.css';

const Quote = () => (
  <h2>
    &quot;Mathematics is not about numbers, equations, computations or algorithms:
    it is about understanding.&quot; -William Paul Thurston
  </h2>
);

export default Quote;
