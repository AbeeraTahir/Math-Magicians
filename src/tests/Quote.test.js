import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

it('Calculator component should render correctly', () => {
  const tree = renderer
    .create(<Quote />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
