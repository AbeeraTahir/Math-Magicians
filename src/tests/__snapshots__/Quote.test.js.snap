// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator component should render correctly 1`] = `
<h2
  className="quote"
>
  "Mathematics is not about numbers, equations, computations or algorithms: it is about understanding." -William Paul Thurston
</h2>
`;
